### PR TITLE
enh(Extension): change Wrapper to typeScript/FuncComponent

### PR DESCRIPTION
**Pull request by Yassir-BenBOUBKER**
…t -Wrapper

## Description

Change styling method from SCSS to makeStyle hook to help in developping the dark mode
change Wrapper/index.js from Class Components to Function Component (ClassComponent doesn't support makeStyle hook)
change the Wrapper to TypeScript file
 
**Fixes** # (issue)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

see if the Administration > Extensions > Manager Page style is the same as before the change
see if the component fonctionality is the same as before the change

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Comment by Yassir-BenBOUBKER** [comment]
replace SCSS module by makeStyles hook in Extensions/Wrappier/index.js
change to TypeScript - FunctionComponent

File: www/front_src/src/Extensions/Wrapper/index.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles({
+  contentWrapper: {
+    '@media (min-width: 767px)': {
```
Comment (Thebarda) [inline www/front_src/src/Extensions/Wrapper/index.tsx:7]: It's better to use the breakpoints provided by Material UI
https://mui.com/customization/breakpoints/#css-media-queries

File: www/front_src/src/Extensions/Wrapper/index.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      padding: '12px',
```
Comment (Thebarda) [inline www/front_src/src/Extensions/Wrapper/index.tsx:8]: We used to use `theme.spacing` from MUI

File: www/front_src/src/Extensions/Wrapper/index.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    boxSizing: 'border-box',
+    margin: '0 auto',
+
+    padding: '12px 20px 0 20px',
```
Comment (Thebarda) [inline www/front_src/src/Extensions/Wrapper/index.tsx:13]: Same comment as above

**Review by Thebarda — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request closed without merging**